proto/src/main.cpp

```diff
@@ -84,6 +85,13 @@ void loop() {
         }
     }
 
+    // If there is valid time pulled from cellular, get time from GPS (if valid)
+    if(Time.now() < 1609459201){
```

Comment:
Is there any way around having this magic value? I assume this is some UTC time (that looks like it maps to Friday, January 1, 2021 12:00:01 AM?

Reply:
Yes, it looks like there are indeed functions to check the validity of the time properly! Done